draft-ietf-mls-protocol.md

```diff
@@ -3361,19 +3361,25 @@ The client verifies the validity of a KeyPackage using the following steps:
 
 # Group Creation
 
-A group is always created with a single member, the "creator".  The other
-members are added when the creator effectively sends itself Add proposals,
-commits them, and then sends the corresponding Welcome message to the new
-participants.  These processes are described in detail in {{add}}, {{commit}},
-and {{joining-via-welcome-message}}.
+A group is always created with a single member, the "creator".  Other members
+are then added to the group using the usual Add/Commit mechanism.
 
-The creator of a group MUST take the following steps to initialize the group:
+The creator of a group is responsible for setting the group ID, ciphersuite, and
+initial extensions for the group.  If the creator intends to add other members
+at the time of creation, then it SHOULD Fetch KeyPackages for the members to be
+added, and select a version and ciphersuite according to the capabilities of the
```

Comment:
> select a version and ciphersuite

In the previous sentence he doesn't choose a version, only ciphersuite. I guess the version is always MLS1.0?